sherpa/astro/datastack/ds.py

```diff
@@ -28,6 +27,10 @@
 
 logger = config_logger(__name__)
 
+try:
+    import stk
+except:
+    logger.warning("could not import stk library. CIAO stack files and syntax will be disabled")
```

Comment:
You are absolutely right, I noticed that too. I have the temptation to fix things like this as I go along, but I don't want that to be in the way of the python 3 migration, and the clock is ticking. There does not seem to be any reason for the duplication, but there is some complexity in the way the wrapping is done, so that might hide a trap... or I just forgot to remove a method when I did the refactoring for #22. Either way, I would rather spend the time required to figure this out adding test coverage where we are missing it. In any case I appreciate the level of your review, so please keep making these kinds of comments :)